doc/index.rst

```diff
@@ -1,55 +1,12 @@
-======
-DWI_ML
-======
-
-.. image:: https://img.shields.io/badge/License-MIT-yellow.svg
-  :target: https://github.com/scil-vital/dwi_ml/blob/master/LICENSE
-
-Welcome to the `Sherbrooke Connectivity Imaging Lab (SCIL)`_ and
-`Videos & Images Theory and Analytics Laboratory (VITAL)`_ joint DWI_ML
-toolkit !
-
-About
-=====
-
-DWI_ML is a toolkit for Diffusion Magnetic Resonance Imaging (dMRI) analysis
-using machine learning and deep learning methods. It is mostly focused on the
-tractography derivatives of dMRI.
-
-**Here is the usual workflow for people using dwi_ml**:
-
-    #. Preprocess your diffusion data using Tractoflow (see :ref:`ref_preprocessing`).
-    #. Preprocess your tractogram using RecobundlesX (see :ref:`ref_preprocessing`).
-    #. Create a new repository for your project. Create a 'scripts_bash' folder and copy our scripts from please_copy_and_adapt. Adapt based on your needs and run:
-
-        #. Run organize_from_tractoflow.sh
-        #. Run organize_from_recobundles.sh
-        #. Run create_dataset.sh
-        #. Run training.sh
-        #. Run tracking.sh
+Welcome to DWI_ML documentation!
+================================
 
 .. toctree::
     :maxdepth: 1
     :caption: Table of contents
 
+    readme_link
```

Comment:
Tested. This works.

Or maybe you could add the toctree but instead of adding a page readme_link, you could write this part` .. include:: ../README.rst` under the toctree. This way, the main page of the website would already be the same as the main page of the github (except the table of contents), and you wouldn't need the readme_link page anymore.

Reply:
Weird. I tried it and it worked. If you think it would be better we can skype to find out why.

Reply:
I spent quite some time trying to make that work on my system without success, so if the current solution works for those trying the PR, then I'd go forward with it. You can still open an issue if you feel like you would like to try and find out a why it would not work on all systems/how to make it work.